R/seq_linter.R

```diff
@@ -9,6 +9,39 @@
 #' These often cause bugs when the right-hand side is zero.
 #' It is safer to use [base::seq_len()] or [base::seq_along()] instead.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "seq(length(x))",
```

Comment:
let's do an n:1 example to show the rev() recommendation?

Reply:
This gives wrong warning message right now. Related to #531 and #1542. So maybe we hold off on including an example? 

``` r
library(lintr)

lint(
  text = "ncol(x):1L",
  linters = seq_linter()
)
#> <text>:1:1: warning: [seq_linter] ncol(...):1L is likely to be wrong in the empty edge case. Use seq_len(1L) instead.
#> ncol(x):1L
#> ^~~~~~~~~~
```

<sup>Created on 2022-10-11 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
OK good point. we can add such an example along with closing the other issues.